roles/dd-nodepool/files/nodepool_server.py

```diff
@@ -55,3 +70,10 @@ def get_nodepool_status(self, status_url, timeout, service_check_tags):
                            AgentCheck.OK,
                            message='Received status from %s' % status_url,
                            tags=service_check_tags)
+
+    def get_nodepool_images(self, images_url, timeout, images_check_tags):
+        r = requests.get(images_url, timeout=timeout)
+        currentTime = int(time.time())
+
+        for image in r.json():
+            self.gauge(self.IMAGES_CHECK_NAME, currentTime - image['age'])
```

Comment:
@jamielennox when I first started taking a look at this, the image age being an absolute time was the first point of contention. If you hop on prod and look at the age of an image, it should be an absolute time, which is why this check has to compare that against the current time to get an accurate value.